var/spack/repos/builtin/packages/mvapich/package.py

```diff
@@ -0,0 +1,300 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os.path
+import re
+import sys
+
+from spack.package import *
+
+
+class Mvapich(AutotoolsPackage):
+    """Mvapich is a High-Performance MPI Library for clusters with diverse
+    networks (InfiniBand, Omni-Path, Ethernet/iWARP, and RoCE) and computing
+    platforms (x86 (Intel and AMD), ARM and OpenPOWER)"""
+
+    homepage = "https://mvapich.cse.ohio-state.edu/userguide/userguide_spack/"
+    url = "https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-3.0a.tar.gz"
+    list_url = "https://mvapich.cse.ohio-state.edu/downloads/"
+
+    maintainers = ["natshineman", "harisubramoni", "ndcontini"]
+
+    executables = ["^mpiname$", "^mpichversion$"]
+
+    # Prefer the latest stable release
+    version("3.0a", "71f6593bfbfe9a9f6f5c750904461f007bf74bec479544e4da375b7d4a56b2ac")
+
+    provides("mpi")
+    provides("mpi@:3.1", when="@2.3:")
+    provides("mpi@:3.0", when="@2.1:")
```

Comment:
Can we simplify these directives? My understanding is that this new package will contain only MVAPICH >= 3.0 right?